trello_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TKN = os.environ.get('TRELLO_TKN')
+
+TRELLO_BOARD = '5f48fa1737fa77134ffe1e17'
+TRELLO_TODO_LIST = '5f48fa17b70abe80f7f0942d'
+TRELLO_DONE_LIST = '5f48fa17d5f15e744c86a725'
+TRELLO_URL_BASE = 'https://api.trello.com/1/'
```

Comment:
You should update the README to explain that this app needs you to have a Trello account and mention these config values.